ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -423,8 +385,10 @@ $(document).ready(function() {
     $("#scheduled-send-clock-selector").change(handleScheduledSendSetupDependingOptions);
 
     $("#synchronize-media-clock-button").click(function() {
-        scheduled_send_media_clock_offset = Date.now();
-        stopResetRunningClock();
+        scheduled_send_media_clock_offset = moment();
+        //stopResetRunningClock();
+        $('#running-clock').hide();
+        $('#running-clock-media').show();
         interval_running_clock = setInterval(updateRunningClockMedia, 500);
```

Comment:
OK well one thing that the function in line 389 does is to clear the current interval before creating a new one!